ops/prod/main.tf

```diff
@@ -23,10 +23,21 @@ resource "azurerm_storage_account" "app" {
   allow_nested_items_to_be_public  = false
   cross_tenant_replication_enabled = false
 
+  queue_properties {
```

Comment:
Not familiar with terraform so I was looking here (https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/data-sources/storage_account) and couldn't find anything regarding `queue_properties` -- can you share what this is for?

Reply:
Sure!

The Terraform link to this `resource` object is [here](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/storage_account); you linked to the `data` object.

This block sets logging rules for the storage queue service.

This block is set to replicate our current settings as closely as possible and introduce as little change as possible. We don't log for the read/write/delete actions on these queues. I've written a ticket to investigate these settings more closely and consider turning them on.

Reply:
Ah ha! Thank you for pointing me in the right direction and the explanation!! 🙌